src/f32-vscaleextexp/f32-vscaleextexp.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef XNN_UKERNEL_WITH_PARAMS
+#define XNN_UKERNEL_WITH_PARAMS(arch_flags, ukernel, element_tile, datatype, params_type, init_params) \
+    XNN_UKERNEL(arch_flags, ukernel, element_tile, datatype)
+#define XNN_DEFINED_UKERNEL_WITH_PARAMS
+#endif
+
+#ifndef XNN_UKERNEL
+#define XNN_UKERNEL(arch_flags, ukernel, element_tile,  datatype) \
+    XNN_UKERNEL_WITH_PARAMS(arch_flags, ukernel, element_tile,  datatype, void, /*init_params=*/nullptr)
+#define XNN_DEFINED_UKERNEL
+#endif
+
+#if XNN_ARCH_X86 || XNN_ARCH_X86_64
+XNN_UKERNEL_WITH_PARAMS(xnn_arch_x86_avx2, xnn_f32_vscaleextexp_ukernel__avx2_p5_u8, 8, float, struct xnn_f32_default_params, xnn_f32_vscaleextexp_ukernel_fn)
```

Comment:
Why is `xnn_f32_vscaleextexp_ukernel_fn` here in the last parameter? I'd expect it to be NULL.